### ISSUE TITLE
A way to start system tray without starting the cluster

### ISSUE THREAD

**Issue by gorkem** [minishift/minishift#3054]
I would like to be able to start the system tray without starting the cluster. I have tried `minishift services systemtray` which starts the systemtray but it is not permanent.

**Comment by gbraad** [comment]
```
$ minishift service systemtray start
```

should actually perform a running process... @anjannath ? Any idea... or does this also exist in proxy? I do not recall. If we need to solve this, let's solve it for all services.

**Comment by anjannath** [comment]
@gbraad Yes, it does, but running this command doesn't start the tray in background, it'll block the terminal the command is run from, in the implementation we run this command but also put it to background by modifying some attributes of the Cmd struct before we run it.

It is the same for the 
```
$ minishift services sftpd
$ minishift services proxy
```

**Comment by LalatenduMohanty** [comment]
Resolved via #3118
